AppServer/google/appengine/api/appcontroller_client.py

```diff
@@ -329,6 +329,19 @@ def add_routing_for_appserver(self, app_id, appserver_ip, port):
       app_id, appserver_ip, port, self.secret)
 
 
+  def remove_routing_for_appserver(self, version_key, ip, port):
+    """ Instructs the AppController to stop routing traffic to an AppServer.
+
+    Args:
+      version_key: A string specifying the version key.
+      ip: A string specifying the location of the AppServer instance.
+      port: An integer specifying the location of the AppServer instance.
```

Comment:
`An integer specifying the port the AppServer instance is using` otherwise its very similar to the other docstring